specification/storagesync/resource-manager/readme.md

```diff
@@ -76,7 +76,7 @@ csharp:
   azure-arm: true
   license-header: MICROSOFT_MIT_NO_VERSION
   namespace: Microsoft.Azure.Management.StorageSync
-  payload-flattening-threshold: 2

```

Comment:
`StorageSync` C# SDK has not been published yet, this won't be a breaking change.
@sergey-shandar Please go ahead and merge this